app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -154,6 +154,7 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
               params: {
                 arrivalAtMempoolEstimate: extraInfo?.arrivalAtMempoolEstimate,
                 status,
+                successAction: paymentDetail?.successAction || null,
```

Comment:
this does not makes sense.. if there are not payment details or successAction then it would be `undefinied || null`